sentry/src/main/java/io/sentry/ISpan.java

```diff
@@ -40,24 +40,20 @@ ISpan startChild(
   SentryTraceHeader toSentryTrace();
 
   /**
-   * Returns the trace state information. @see <a
-   * href="https://develop.sentry.dev/sdk/trace-context/">Trace Context</a>.
+   * Returns the trace context.
    *
-   * @return a trace state or {@code null} if {@link SentryOptions#isTraceSampling()} is disabled.
+   * @return a trace context or {@code null} if {@link SentryOptions#isTraceSampling()} is disabled.
    */
   @Nullable
-  @ApiStatus.Experimental
```

Comment:
Should the experimental markers stay for any baggage / trace context related stuff?

Reply:
They should stay according to a discussion on slack.